test/ScanForErrorMiddleware/ErrorMiddlewareFilterTest.php

```diff
@@ -10,6 +10,7 @@
 use ArrayIterator;
 use DirectoryIterator;
 use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
```

Comment:
Same question here: it's not used anywhere, and tests pass without importing it...